eks-worker-al2.json

```diff
@@ -114,34 +115,34 @@
   "provisioners": [
     {
       "type": "shell",
-      "remote_folder": "{{ user `remote_folder`}}",
-      "script": "{{template_dir}}/scripts/install_additional_repos.sh",
-      "environment_vars": [
-        "ADDITIONAL_YUM_REPOS={{user `additional_yum_repos`}}"
+      "inline": [
+        "mkdir -p {{user `working_dir`}}",
+        "mkdir -p {{user `working_dir`}}/log-collector-script"
```

Comment:
Can't this just be combined into one line?

Reply:
Yeah. I think I separated this to make it very obvious what the intent of this provisioner is, I don't feel too strongly either way.

Reply:
Ya. I had mixed feelings. It's unnecessary, but removes the illusion that we're creating this just for the log collector script.